src/Umbraco.Web/PublishedCache/NuCache/DataSource/ContentData.cs

```diff
@@ -6,7 +6,7 @@ namespace Umbraco.Web.PublishedCache.NuCache.DataSource
     /// <summary>
     /// Represents everything that is specific to an edited or published content version
     /// </summary>
-    internal class ContentData
+    public class ContentData
```

Comment:
need to be public?

Reply:
Yes, in order to implement IDataSource, ContentNode needs to be public. I'm open to ideas on how to refactor ContentData to make it less weird, not really sure how to improve.